builtin/logical/pki/backend.go

```diff
@@ -408,6 +414,13 @@ func (b *backend) periodicFunc(ctx context.Context, request *logical.Request) er
 		return err
 	}
 
+	// If a delta CRL was rebuilt above as part of the complete CRL rebuild,
```

Comment:
We might need to check if we are running on the active node within periodicFunc?